src/features/landing/components/AboutUs/DeviceImage/styles.ts

```diff
@@ -31,7 +31,7 @@ export const deviceImageStyles = {
     borderTopRightRadius: '30px',
     borderTopLeftRadius: '30px',
     overflow: 'hidden',
-    '@media (max-width: 1024.98px)': {
+    '@media (max-width: 1023.98px)': {
```

Comment:
this looks strange, need use 1024

Reply:
if I make a 1024, it won't look like it does on the figma